### PR TITLE
clean up changelog, metric docs

### PR DESCRIPTION
**Pull request by cartersocha**
Cleaned up metric docs

Added additional changelog updates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,13 +23,19 @@ significant modifications will be credited to OpenTelemetry Authors.
 ([#141](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/141))
 * Added additional attributes to productcatalog service
 ([#143](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/143))
-* Added manual instrumentation to Java ad service
+* Added manual instrumentation to ad service
 ([#150](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/150))
 * Added manual instrumentation to Ruby email service
```
Comment (reyang) [inline CHANGELOG.md:28]: Following that change on L26, is the intention to remove "Ruby" here as well?

File: docs/metric_service_features.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+## OpenTelemetry Feature Set by Services
+
+### Emoji Legend
+
+- Completed: :100:
+- Not Present (Yet): :construction:
+
+### [Ad Service](../src/adservice/README.md)
+
+tba
```
Comment (reyang) [inline docs/metric_service_features.md:26]: Minor, normally I've seen folks putting "TBD" instead of "tba"

**Review by reyang — APPROVED** [review]

File: docs/metric_service_features.md (hunk 2 above)
Comment (cartersocha) [reply docs/metric_service_features.md:26]: In my mind it’s already decided we’ll add the info. Just not there yet 😉 but I’ll update

File: CHANGELOG.md (hunk 1 above)
Comment (cartersocha) [reply CHANGELOG.md:28]: Removed

**Review by wph95 — APPROVED** [review]

**Pull request merged**